src/api-specification/peer-controller/retrieving-peer-limit-status.md

```diff
@@ -1,5 +1,7 @@
 # Retrieving peer limit status
 
+> Retrieves the limit status of the [authorized FIB user](../auth-controller/authorizing-a-user-in-the-system.md) at the time of the request.
```

Comment:
status is kinda 'exceeded/not' value to me, how about reformatting that to `state` instead?